config/configcompression/compressiontype.go

```diff
@@ -17,8 +17,15 @@ const (
 	empty   CompressionType = ""
 )
 
+// IsCompressed Deprecated [0.94.0] use member function CompressionType.IsCompressed instead
```

Comment:
According to https://go.dev/blog/godoc the deprecation comment should be on its own line and start with `Deprecated:`
```suggestion
// IsCompressed returns false if CompressionType is nil, none, or empty. Otherwise it returns true.
//
// Deprecated: [0.94.0] use member function CompressionType.IsCompressed instead
```